### PR TITLE
Fixed #28 Create documentation of how to generate auto tests from Cap…

### LINKED ISSUE #28: Create documentation of how to generate auto tests from Capella model
Dedicated section to be added in the Python4Capella Developer Manual

### PR DESCRIPTION
**Pull request by ylussaud**
…ella model.

### CODE REVIEW

File: doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile [hunk 1]
```diff
@@ -9,3 +9,4 @@ h2. Advanced Python users
 h2. Python4Capella contributors
 
 * "*Create a Python interpreter distribution for Windows*":python/CreateWindowsPythonFile.html Describes how you can recreate the Python interpreter zip file for Windows.
+* "*Generate Python tests*":python/GeneratePythonTests.html Describes how you can Python tests for the Python Capella light metamodel.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile:12]: A verb is missing "how you can write/add..."

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+h1. Generate Python Test
+
+This section describes how you can generate Python tests for the simplified Capella Python metamodel.
+
+The generator is defined in the "ProduceCapellaPythonTestsFromCapellaHandler":https://github.com/labs4capella/python4capella/blob/master/plugins/org.eclipse.python4capella.gen/src/org/eclipse/python4capella/gen/handlers/ProduceCapellaPythonTestsFromCapellaHandler.java Java class. It use the simplified Capella metamodel defined in the "specification folder":https://github.com/labs4capella/python4capella/tree/master/specification of the git repository.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:5]: It "uses"

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+The Capella Light Metamodel is defined using Capella 5.0, you will need this version to generate the Python tests.
+
+Start to import the "org.eclipse.python4capella.gen":https://github.com/labs4capella/python4capella/tree/master/plugins/org.eclipse.python4capella.gen project in your Capella 5.0 workspace and "enable developer capabilities":../../user/general/EnableDevelopmentCapabilities.html to launch a runtime:
+
+To open the specification model, you will need import the project in the workspace of your runtime.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:11]: need to

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+When you opened the specification model, you can open the System Analysis, then the Data folder, then select the Capella Light Metamodel. You can right click on it and select the "Generate Python Tests from Capella" menu:
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:13]: you open

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+!GeneratePythonTests/GeneratePythonTestsMenu.png(Generate Python Tests from Capella menu)!
+
+By default this will generate a file "/tmp/capella/capella_tests.py". You can change this path by editing the generator class: ProduceCapellaPythonTestsFromCapellaHandler.
+
+For the moment, the only customization you can add is mark a test as read only. This will generate a test that only access a getter for this test and will not make use of any setter.
```
Comment (mbats) [inline doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:19]: is to

**Review by mbats — APPROVED** [review]

File: doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile (hunk 1 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile:12]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile (hunk 3 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:11]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile (hunk 2 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:5]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile (hunk 4 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:13]: done

File: doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile (hunk 5 above)
Comment (ylussaud) [reply doc/org.eclipse.python4capella.doc/doc/developer/python/GeneratePythonTests.textile:19]: done

**Pull request merged**